.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Apply php-cs-fixer fix --rules nullable_type_declaration_for_default_null_value
+f4118e110a46de3ffb799e7d79bf15128d1646ea
```

Comment:
:open_mouth:

Reply:
See https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view ;)